periphery/nonfungible_position_manager.go

```diff
@@ -44,12 +44,12 @@ type IncreaseSpecificOptions struct {
 	TokenID *big.Int // Indicates the ID of the position to increase liquidity for
 }
 
-//  Options for producing the calldata to add liquidity
+// Options for producing the calldata to add liquidity
 type CommonAddLiquidityOptions struct {
-	SlippageTolerance *core.Percent  // How much the pool price is allowed to move
-	Deadline          *big.Int       // When the transaction expires, in epoch seconds
-	UseNative         *core.Currency // Whether to spend ether. If true, one of the pool tokens must be WETH, by default false
-	NativeToken       *core.Token    // TODO: merge this with UseNative
+	SlippageTolerance *core.Percent      // How much the pool price is allowed to move
+	Deadline          *big.Int           // When the transaction expires, in epoch seconds
+	UseNative         *core.BaseCurrency // Whether to spend ether. If true, one of the pool tokens must be WETH, by default false
```

Comment:
BaseCurrency isn't available in v0.1.5, as far as I check

https://pkg.go.dev/github.com/daoleno/uniswap-sdk-core@v0.1.6/entities
https://pkg.go.dev/github.com/daoleno/uniswap-sdk-core@v0.1.5/entities